server/leafnode.go

```diff
@@ -108,8 +108,59 @@ func (c *client) isHubLeafNode() bool {
 	return c.kind == LEAF && !c.leaf.isSpoke
 }
 
+// Will add in the deny exports and imports for JetStream on solicited connections if we
+// are sharing the system account and wanting to extend the JS domain.
+func (s *Server) addInJSDeny(r *RemoteLeafOpts) {
+	var hasDE, hasDI bool
+	for _, dsubj := range r.DenyExports {
+		if dsubj == jsAllApi {
+			hasDE = true
+			break
+		}
+	}
+	for _, dsubj := range r.DenyImports {
+		if dsubj == jsAllApi {
+			hasDI = true
+			break
+		}
+	}
+	if !hasDE {
+		s.Warnf("Adding deny export of %q for leafnode configuration on %q that bridges system account", jsAllApi, r.LocalAccount)
```

Comment:
Info instead of warning? (same for warn below)

Reply:
Sure.. Will make the change.